cmd/dagger/engine.go

```diff
@@ -52,19 +64,27 @@ func withEngineAndTUI(
 		engineConf.JournalFile = os.Getenv("_EXPERIMENTAL_DAGGER_JOURNAL")
 	}
 
-	if silent {
-		if engineConf.LogOutput == nil {
-			engineConf.LogOutput = os.Stderr
+	if !silent {
+		if progress == "auto" && autoTTY || progress == "tty" {
+			if interactive {
+				return interactiveTUI(ctx, engineConf, fn)
```

Comment:
Do we not set the `EngineName` and `CloudURL` callbacks for the interactive TUI? If so, is that just an oversight or is it because we'll need more updates to display those in the interactive tui? Not a big deal either way, just checking

Reply:
Needs more updates; will probably do it in another PR as this one's already grown quite a bit.